src/helpers/AvroToMetaModel.ts

```diff
@@ -0,0 +1,435 @@
+import {
+  AnyModel,
+  ArrayModel,
+  AvroSchema,
+  BooleanModel,
+  EnumModel,
+  EnumValueModel,
+  FloatModel,
+  IntegerModel,
+  MetaModel,
+  MetaModelOptions,
+  ObjectModel,
+  ObjectPropertyModel,
+  StringModel,
+  UnionModel
+} from '../models';
+import { Logger } from '../utils';
+
+function getMetaModelOptions(AvroModel: AvroSchema): MetaModelOptions {
```

Comment:
I would definitely add some tests for this file to make it easier to assess how it behaves :v:

Reply:
Tests are in progress and would push that by today.